### DISCUSSION TITLE
Error in Downsampling

### DISCUSSION THREAD

**Discussion by DaniyalBME** [bachlab/PsPM#782] [Q&A] [answered]
Dear PSPM Team,

I hope this message finds you well.

I am writing to get your help in downsampling my data to 10Hz, as I’ve encountered an error in the process (please see the attached screenshots). I would greatly appreciate it if you could help me understand why this error is occurring. I’ve also attached the data file I was working with.

Thank you very much for your time and assistance. Please let me know if you need any additional information.

Best regards,
Daniyal

![error_downsampling](https://github.com/user-attachments/assets/b3ac53b3-66a5-4f0c-be9f-c3f7e46187dc)

[Uploading PD_01.zip…]()

### Technical Info
* PsPM revision ID: Version 6.1.2
* MATLAB version:R2023b
* ADInstrument Skin Conductance Recording Device

**Comment by dominikbach** [comment, marked as answer]
Hi Daniyal. We are discontinuing the function `pspm_down` and have now moved the functionality to `pspm_filter`. Could you clone the develop version of the repository and try to run your job with the "Data filtering" GUI item, selecting the "Butterworth filter" and disabling high-pass and low-pass?

**Comment by DaniyalBME** [comment]
Dear Dominik,
Thank you for your time and the update. I’ve tried the data filtering with the "Butterworth filter" as suggested, and it's working perfectly now.

Once again, thank you very much for your help!

Best regards,
Daniyal
